File: FashionMnist/app/src/main/java/com/example/fashionmnist/MainActivity.java
package com.example.fashionmnist;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.graphics.Bitmap;
import android.graphics.Color;
import android.os.Bundle;
import android.util.Pair;
import android.widget.Button;
import android.widget.TextView;

import com.divyanshu.draw.widget.DrawView;
import com.example.fashionmnist.draw.DrawActivity;
import com.example.fashionmnist.tflite.Classifier;

import java.util.Locale;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        Button drawBtn = findViewById(R.id.drawBtn);
        drawBtn.setOnClickListener(view -> {
            Intent i = new Intent(MainActivity.this, GalleryActivity.class);
            startActivity(i);
        });
    }
}

File: Activity & Intent/Save_Personal_Info/app/src/main/java/com/example/save_personal_info/MainActivity.kt
package com.example.save_personal_info

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.example.save_personal_info.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {

    val binding by lazy  { ActivityMainBinding.inflate(layoutInflater) }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(binding.root)

        with(binding){
            Mbtn.setOnClickListener{
                var infoData = arrayOf(Mname.text.toString(), Mage.text.toString(), Mtel.text.toString(),
                    Maddr.text.toString(), Metc.text.toString())

                val intent = Intent(root.context, SubActivity::class.java)
                intent.putExtra("Info_from_Main", infoData)
                startActivity(intent)
            }
        }
    }
}